src/core/CombinedTargetAndLanguageLevel.ts

```diff
@@ -0,0 +1,28 @@
+import { ScriptTarget } from './File';
+
+export class CombinedTargetAndLanguageLevel {
+
+	constructor(private combination: string) {
+		this.combination = this.combination || "browser";
+	}
+
+	public target(): string {
+		const [target,] = this.splitCombination();
+		return target;
+	}
+
+	public languageLevel(): ScriptTarget {
```

Comment:
One last thing, do you think you can add getters or change these method names to `getLanguageLevel` like everywhere in the code base? It seems a little bit inconsistent.